packages/subscriber/src/components/add-form/index.tsx

```diff
@@ -133,103 +140,107 @@ export const AddSubscriberForm: FunctionComponent< Props > = ( props ) => {
+				<Title>Add subscribers to build your audience</Title>
+			</div>
+
+			<div className={ 'add-subscriber__form--container' }>
+				{ inProgress && <Notice isDismissible={ false }>You have email list importing...</Notice> }
+
+				<form onSubmit={ onFormSubmit }>
+					<TextControl
+						placeholder={ 'sibling@email.com' }
+						value={ emails[ 0 ] || '' }
+						help={ isValidEmails[ 0 ] ? <Icon icon={ check } /> : undefined }
+						onChange={ ( value ) => onEmailChange( value, 0 ) }
+					/>
+					<TextControl
+						placeholder={ 'parents@email.com' }
+						value={ emails[ 1 ] || '' }
+						help={ isValidEmails[ 1 ] ? <Icon icon={ check } /> : undefined }
+						onChange={ ( value ) => onEmailChange( value, 1 ) }
+					/>
+					<TextControl
+						placeholder={ 'friend@email.com' }
+						value={ emails[ 2 ] || '' }
+						help={ isValidEmails[ 2 ] ? <Icon icon={ check } /> : undefined }
+						onChange={ ( value ) => onEmailChange( value, 2 ) }
+					/>
+
+					{ ! isSelectedFileValid && (
+						<label className={ 'add-subscriber__form-label-error' }>
+							{ createInterpolateElement(
+								__(
+									'<span><icon /> Sorry, you can only upload a CSV file.</span><uploadBtn>Select another file</uploadBtn>'
+								),
+								{
+									span: createElement( 'span' ),
+									icon: createElement( Icon, { icon: info, size: 20 } ),
+									uploadBtn: formFileUploadElement,
+								}
 							) }
-
-							{ isSelectedFileValid && selectedFile && (
-								<label className={ 'add-subscriber__form-label-links' }>
-									{ createInterpolateElement(
-										sprintf(
-											/* translators: the first string variable shows a selected file name, Replace and Remove are links" */
-											__(
-												'<strong>%s</strong> <uploadBtn>Replace</uploadBtn> | <removeBtn>Remove</removeBtn>'
-											),
-											selectedFile?.name
-										),
-										{
-											strong: createElement( 'strong' ),
-											uploadBtn: formFileUploadElement,
-											removeBtn: createElement( Button, {
-												isLink: true,
-												onClick: () => setSelectedFile( undefined ),
-											} ),
-										}
-									) }
-								</label>
-							) }
-
-							{ isSelectedFileValid && ! selectedFile && (
-								<label>
-									{ createInterpolateElement(
-										__(
-											'Or bring your mailing list from other newsletter services by <uploadBtn>uploading a CSV file.</uploadBtn>'
-										),
-										{ uploadBtn: formFileUploadElement }
-									) }
-								</label>
+						</label>
+					) }
+
+					{ isSelectedFileValid && selectedFile && (
+						<label className={ 'add-subscriber__form-label-links' }>
+							{ createInterpolateElement(
+								sprintf(
+									/* translators: the first string variable shows a selected file name, Replace and Remove are links" */
+									__(
+										'<strong>%s</strong> <uploadBtn>Replace</uploadBtn> | <removeBtn>Remove</removeBtn>'
```

Comment:
Why should you keep these together in one translation string, and not separately?

Reply:
I'm trying to remember, but the first version of that translation was a sentence with one 'Remove' link. After some time, we got a new design with two links, and I just updated it; that's why it came to this version, I guess.